unreleased/tql-limit-func-priority-keys.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Comment:
Since we don't have a way to default values in a function call this should be a breaking change.

Reply:
It is still possible to use the `limit` function with a field and the limit, e.g. `limit(attrs, 5)` without passing priority keys, that would result in the same behaviour as it before this change, so it doesn't really break anything for the end user.

Reply:
It would seem the TQL function caller defaults slice type parameter to an empty slice.  I'm not entirely sure we intended that and should encourage that behavior, but its fine in this instance I guess.

Reply:
So we do not change the change log type for now? Do I need to do anything on this PR?